src/com/company/Main.java

```diff
@@ -1,8 +1,14 @@
 package com.company;
 
+import java.util.List;
+
 public class Main {
 
     public static void main(String[] args) {
-	// write your code here
+        EvenNumbersGenerator evenNumbersGenerator = new EvenNumbersGenerator();
+        List<Integer> result = evenNumbersGenerator.getEvenNumbersList(0, 100);
+        for ( int temp : result ) {
```

Comment:
pls reformat code

Reply:
this for loop can be written 

this way
```
            result.forEach(each -> System.out.println(each));
```

or this 
```
result.forEach(System.out::println);
```

Reply:
Done